### ISSUE TITLE
Rule idea: function-no-unknown

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1779]
Valid:
```css
a {
  filter: blur(10px);
}
```

Invalid:
```css
a {
  filter: blurr(10px);
  color: rgbaa(10, 10, 10, 1);
}
```

**Comment by jeddy3** [comment]
[`css-values`](https://github.com/ben-eb/css-values) is gathering momentum and I believe it'll make these `*-no-unknown` rules redundant. 

Perhaps best to draw a line under what we have at the moment and focus on contributing to `css-values`?

I'm easy on this one though.

**Comment by evilebottnawi** [comment]
@jeddy3 By creating a rule we get feedback from users and accelerate the development of the project

**Comment by jeddy3** [comment]
> accelerate the development of the project

Is that our goal? 

Slowly helping out with `css-values` project means it could be ready in time for `8.0.0` - making our existing `no-unknown` rules redundant. Combining forces, rather than accelerating the development of stylelint, feels like a better trajectory to me.

**Comment by evilebottnawi** [comment]
@jeddy3 yep, you are right

**Comment by Mouvedia** [comment]
followup: #5853
